uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/AzureSampleApp.java

```diff
@@ -93,6 +95,13 @@ public void signInWithSingleAccountFragment(@NonNull final String username,
                 new MicrosoftStsPromptHandler(promptHandlerParameters);
 
         microsoftStsPromptHandler.handlePrompt(username, password);
+
+        // sleep as it can take a bit for UPN to appear in Azure Sample app
```

Comment:
Seems to be taking a lot more than "a bit" with a sleep of 10 seconds 😁

Reply:
I tested knocking this down to 5 seconds and seems like it still boosts reliability as needed.

Reply:
Okay, do you want to change it to 5 seconds in the code then?